README.md

```diff
@@ -8,11 +8,24 @@ Azadi is an SoC with 32-bit RISC-V single core "[lowrisc/ibex](https://github.co
 Ibex+FPU passes all the compliance test of RV32IMFC, here is the [status](https://docs.google.com/spreadsheets/d/1gIzSU5mb4L3pPdiJr7MkdhvupT7p5VF2qy1PzDwq-5I/edit#gid=862473485) of tests.
 The status of all the tests can be found [Verification Basic Tests Sheet.](https://docs.google.com/spreadsheets/d/1gIzSU5mb4L3pPdiJr7MkdhvupT7p5VF2qy1PzDwq-5I/edit#gid=1374860298)
 
-## Quickstart guide
+## Prerequisite 
+==> You need to have the `verilator` , `GtkWave` installed for simulation.
 
-1. Clone the repository
-2. You can build the simulation for verilator by simply running `make` from the root of this repo. You need to have the `verilator` installed for this.
-3. After successful make you can find the waveform file in `logs` folder on root of this repo. To see the waves you should have `gtkwave` installed.
+## Quickstart guide
+1. Clone the repository.
+2. ` cd azadi-II/ `
+3. ` export AZADI_ROOT= pwd `
+4. ` fusesoc library add azadi https://github.com/Nameer-Iqbal-Ansari/azadi-II `
+5. For direct simulation through verilator simply execute `make` from the root of this repo. OR To use fusesoc then execute `fusesoc run --target=sim azadi --cycles=519800 --HEX=$AZADI_ROOT/verif/tests/basic_test/test.hex`
+6. After simulation is done, you can find the waveform file in `$AZADI_ROOT/build/azadi_1.0/sim-verilator/logs`.
 
 ### Block Diagram of SoC
 ![](docs/images/Azadi%20MicroArchitechtureDiagram-SoC.drawio.png)
+### Directory Structure Diagram of SoC
+![](docs/AZADI-II%20directory%20structure.png)
+
+## Issues and Solutions
```

Comment:
Let's keep this issue for a a while here, once the PR is merged then we can move this issue to the issue tracker.

Reply:
Sure.